include/z64object.h

```diff
@@ -2,6 +2,30 @@
 #define Z64OBJECT_H
 
 #include "stdint.h"
+#include "romfile.h"
+
+struct GameState;
+
+typedef struct {
+    /* 0x00 */ s16 id; // Negative ids mean that the object is unloaded
+    /* 0x02 */ UNK_TYPE1 pad2[0x2];
```

Comment:
```suggestion
```
This should just match if you remove it, shouldn't it?

Reply:
This is completely irrelevant to the PR though

Reply:
i know but it's right there